### ISSUE TITLE
Calendar: whole day events appear as 00:00-23:59 events

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#271]
```
What steps will reproduce the problem?

Create a whole day event. When creating the event, navigate with TAB through 
the various fields, namely Start and End without modifying a thing on Start and 
End fields. Click Save. 

What is the expected output? What do you see instead?

The recently added whole day event should appear on the top as that is the 
right place for those events. Instead it appears as a 00:00 to 23:59 event 
occupying the whole day shift.

PS: Just tested it on mail4us.net and results are the same.

What version of the product are you using? On what operating system?

- Using non official version of the bundle (1.8++, for RC-0.6, the calendar.zip 
that corrected  issue 259 )
- Roundcube Webmail 0.6
- Linux, Fedora Core 14
- Davical 0.9.9.7

  Thanks in advance for your time.

Regards

```

Original issue reported on code.google.com by `cpsoa...@gmail.com` on 27 Nov 2011 at 2:24

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 27 Nov 2011 at 5:21
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Attached 'calendar.callbacks.js' will fix the issue (Notice, it is for 0.6 - 
bundle 2.0).
```

Original comment by `rol...@liebl.ath.cx` on 27 Nov 2011 at 6:04

Attachments:
- [calendar.callbacks.js](https://storage.googleapis.com/google-code-attachments/myroundcube/issue-271/comment-2/calendar.callbacks.js)

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 2 Dec 2011 at 6:09
- Changed state: **Fixed**
